OptimizelySDK/Bucketing/DecisionService.cs

```diff
@@ -67,17 +67,8 @@ public DecisionService(Bucketer bucketer, IErrorHandler errorHandler, ProjectCon
         /// <param name = "userId" > The userId of the user.
         /// <param name = "filteredAttributes" > The user's attributes. This should be filtered to just attributes in the Datafile.</param>
         /// <returns>The Variation the user is allocated into.</returns>
-        public Variation GetVariation(Experiment experiment, string userId, UserAttributes filteredAttributes)
+        public virtual Variation GetVariation(Experiment experiment, string userId, UserAttributes filteredAttributes)
```

Comment:
is user profile service not checked in decision service?

Reply:
User Profile service has been checked inside GetVariation method. Please see this for reference:
https://github.com/msohailhussain/csharp-sdk/blob/7026f3f4fb8c4433352a9e9d8dd681e3111081a3/OptimizelySDK/Bucketing/DecisionService.cs#L83